lua/lint/linters/chktex.lua

```diff
@@ -1,37 +1,35 @@
 local pattern = '(%d+):(%d+):(%d+):(.+):(%d+):(.*)'
 local severities = {
-    Error = vim.lsp.protocol.DiagnosticSeverity.Error,
-    Warning = vim.lsp.protocol.DiagnosticSeverity.Warning,
-    Message = vim.lsp.protocol.DiagnosticSeverity.Information
+  Error = vim.lsp.protocol.DiagnosticSeverity.Error,
+  Warning = vim.lsp.protocol.DiagnosticSeverity.Warning,
```

Comment:
Please revert the unrelated formatting changes

Reply:
Sorry about that, I made a small change to the file and reformatted everything.
Do you currently use a formatter so that I can adapt to the same style ?

Reply:
I tend to use the formatting that you would've applied, but so far it is not enforced which is why different parsers have a different style. I wouldn't mind the reformat itself - what I dislike is that it reformats *and* makes a change within the same commit.